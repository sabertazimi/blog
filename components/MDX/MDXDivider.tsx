import Divider from '@/components/Divider'

function MDXDivider(): JSX.Element {
  return <Divider className="bg-gradient-primary h-1 rounded-full" />
}

export default MDXDivider
