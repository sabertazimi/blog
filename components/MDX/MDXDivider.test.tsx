import { render } from '@testing-library/react';
import MDXDivider from './MDXDivider';

describe('MDXDivider', () => {
  test('should render correctly (snapshot)', () => {
    const { container } = render(<MDXDivider />);

    expect(container).toMatchSnapshot();
  });
});
