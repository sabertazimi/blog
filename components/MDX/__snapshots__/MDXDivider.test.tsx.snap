// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MDXDivider should render correctly (snapshot) 1`] = `
<div>
  <div
    class="ant-divider ant-divider-horizontal mx-0 my-12 h-1 rounded-full bg-gradient-primary"
    role="separator"
  />
</div>
`;
